commune_app/all_models/chores.py

```diff
@@ -22,3 +22,8 @@ def description_snippet(self):
     @staticmethod
     def get_chore(id):
         return Chore.objects.filter(id=id).first()
+
+    def execute_chore(self, chore_id,user_id):
+        chore = Chore.objects.filter(id=chore_id)[0]
```

Comment:
maybe you can do it like:
```suggestion
        chore = Chore.objects.filter(id=chore_id).first()
```